src/function.py

```diff
@@ -408,6 +458,18 @@ def create_request():
     return create_request
 
 
+def _set_console_logging_level():
+    """
+    Determines whether or not debug logging should be enabled based on the env var.
+    Defaults to false.
+    """
+    if _debug_logging_enabled():
+        logger.setLevel(logging.DEBUG)
+        logger.debug("Enabled debug mode")
+    else:
+        logger.setLevel(logging.INFO)
```

Comment:
The default level is warn. Making this the default will make this function pretty noisy.

Reply:
All the previous `print()` statements have been converted into `logger.info()` statements, so the logged information will remain the same. The most notorious example being:

```
logger.info("Log entry sent. Response code: {}. url: {}".format(status, url))
```

I could convert this statement to `logger.warn()` instead, but OTOH this would raise concerns from customers thinking that something is wrong. What are your thoughts on this?

Reply:
On second thought I agree, INFO is fine as a default.

Reply:
Thanks for considering it :-)